### ISSUE TITLE
Single switch for Presence Lock

### ISSUE THREAD

**Issue by webjib** [homebridge-plugins/homebridge-tado#82]
Hello! First of all, congratulations for this wonderful plugin!

I'm using the Presence Lock for running automation (exactly like the official Tado option). When going away, I'm turning the Away to On and the Home to Off. And when coming back home, the opposite.

However, it would be great to have an option with a single Switch, instead of a double Switch. Personally, disabling the Presence Lock is useless (and Tado does not offer such an option, there's only "Home" or "Away" modes). Would it be possible to add this as an option (in addition to the Alarm type, and double Switch type)? This would be faster to change the state in the Home app. For example, a "At Home" switch: when ON, it means you're at home. When off, it means you're away. Or a "Away" switch, whatever.

BTW, at first I was using the Presence Lock as an Alarm. It's great because we have a switch with 3 states. Unfortunately, it's not really possible to use it in an automation because you need to confirm the state change everytime (HomeKit displays a notification). This makes sense for a real Alarm, because you don't want the Alarm to get disabled, just because you were near your Home ;-)

**Comment by nielsth** [comment]
I would really appreciate a single switch as well. 

Either by choosing in the config which of the current two (Home/Auto or Away/Auto) to display in HomeKit, or by specifying in the config which modes the single switch should bounce between in its On and Off states (choose one from Home/Away/Auto for both states)

**Comment by UKenGB** [comment]
Yup, 2 switches is a terrible idea as it apparently allows both Home and Away to be set simultaneously which is logically nonsense. Maybe the plug-in automatically turns the other off to avoid this? I don't know for sure as I changed to the 'Alarm' type of representation which is a 3 way switch and avoids the problem, although one option is 'Off' rather than 'Auto' as it should be.

I suspect this is a limitation of HomeKit and Apple really need to pull their socks up with regard to HK generally, but trying to use 2 independent 2 way switches to represent a 3 way option, even if improper selections are prevented, is a really bad interface design.

Apple have had 8 years to improve HomeKit. Must try harder.
